static/js/containers/LearnerSearchPage.js

```diff
@@ -76,15 +80,15 @@ class LearnerSearchPage extends React.Component {
   };
 
   closeEmailComposerAndSend = (): void => {
-    const { dispatch, searchResultEmail: { inputs, params } } = this.props;
+    const { dispatch, searchResultEmail: { inputs } } = this.props;
     let errors = emailValidation(inputs);
     dispatch(updateEmailValidation({type: SEARCH_EMAIL_TYPE, errors: errors}));
     if (R.isEmpty(errors)) {
       dispatch(
         sendSearchResultMail(
           inputs.subject || '',
           inputs.body || '',
-          params.searchkit.buildQuery().query
+          this.searchkit.buildQuery().query
```

Comment:
can you roll back the changes in `openEmailComposer` and `closeEmailComposerAndSend`? PR https://github.com/mitodl/micromasters/pull/2555 heavily refactors the way that the email dialog is defined and invoked. rolling these changes back will make the merge easier, and AFAIK these changes aren't essential for proper functionality